reactivesocket-tck-drivers/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+java -cp build/libs/reactivesocket-tck-drivers-0.2.2-SNAPSHOT.jar io.reactivesocket.tckdrivers.main.Main $@
```

Comment:
For scripts the convention is "$@" which keeps the same arguments that were passed in.  I know its counter intuitive, since you expect it to munge them into one.

Try using: https://www.shellcheck.net/ or install with "brew install shellcheck"